### ISSUE TITLE
Antergos as a VMware guest

### ISSUE THREAD

**Issue by starks** [Antergos/antergos-iso#29]
I can't for the life of me get VM detection (open-vm-tools?) to work properly.

When I deploy Antergos in a VMware environment, most of the guest functions work as expected, but I have never once gotten cursor capture to work.

What Antergos does: Cursor is locked to VM unless I release
What every other distro (including Manjaro) does: Cursor transitions seamlessly to and from VM and host/vcenter

Are the proper drivers and kernel modules not present by default?

**Comment by karasu** [comment]
Hi,

We use virtualbox, so never tested it with vmware.

We can't help a lot with this issue, but you can check that you have these packages installed:
xf86-input-vmmouse
xf86-video-vmware
and open-vm-tools of course.

**Comment by starks** [comment]
"We use virtualbox, so never tested it with vmware."

That's not very encouraging. I'd like to use Antergos at work, but this is a dealbreaker. I really have no good options for quick Arch installs. Manjaro isn't an option since it lacks ABS and has questionable design decisions.

And yes, I have tested packages. I've been trying to get this right for months.

**Comment by starks** [comment]
I wonder if the problem is inherent to GNOME 3 itself. I'm suddenly noticing a VM-crashing pattern with distros that boot to it.

**Comment by dunestorm333** [comment]
@starks You are right, VMware is the most popular and best hyper-visor in the world. The developers should really take time to test their software in VMware.

I can tell you @starks that installing the packages @karasu listed above is enough to make Antergos run well as a host:
- xf86-input-vmmouse
- xf86-video-vmware
- open-vm-tools
